src/temp/ProjectInfo/ContactDetails.tsx

```diff
@@ -0,0 +1,90 @@
+import LocationIcon from '../icons/LocationIcon';
+import MailIcon from '../icons/MailIcon';
+import RightArrowIcon from '../icons/RightArrowIcon';
+import { ViewProfileIcon } from '../icons/ViewProfileIcon';
+import WebsiteLinkIcon from '../icons/WebsiteLinkIcon';
+import styles from './ProjectInfo.module.scss';
+import { useTranslation } from 'next-i18next';
+import SingleProjectInfoItem from './SingleProjectInfoItem';
+
+interface Props {
+  websiteURL: string;
+  location: string;
+  email: string;
+}
+
+const ContactDetails = ({ websiteURL, location, email }: Props) => {
+  const { t } = useTranslation(['donate']);
+
+  const contactDetails = [
+    {
+      icon: (
+        <ViewProfileIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: t('donate:viewProfile'),
+      link: '',
+    },
+    {
+      icon: (
+        <WebsiteLinkIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: websiteURL
+        .replace('https://www.', '')
+        .replace('http://', '')
+        .replace('https://', '')
+        .replace('https://www.', '')
+        .split(/[/?#]/)[0],
+      link: websiteURL,
+    },
+    {
+      icon: (
+        <LocationIcon width={9.5} color={`${'var(--primary-font-color)'}`} />
+      ),
+      title: location,
+      link: '',
+    },
+    {
+      icon: <MailIcon width={10.5} color={`${'var(--primary-font-color)'}`} />,
+      title: email,
+      link: `mailto:${email}`,
+    },
+  ];
+
+  return (
+    <div className={styles.contactDetailsContainer}>
+      <SingleProjectInfoItem
+        title={t('donate:contactDetails')}
+        itemContent={
+          <div className={styles.contactText}>
+            {contactDetails.map((contact, index) => (
+              <a
+                href={contact.link}
```

Comment:
Using the index of the array as the key when mapping through contactDetails might lead to unexpected behavior if the array's items change position or if items are added or removed. It's generally safer to use a unique identifier if available.

Reply:
This would not be the case as this is a static array